CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+We welcome all contributions!
+
+Please, feel free to start working on any issues we have created.
+For starters, we recommend to pick up issues labelled *good first issue*.
+
+In addition to everything written in here, there is a [Collaborator Guide](https://github.com/chingu-voyages/ChinguResourceList/blob/development/docs/COLLABORATOR_GUIDE.md) you can read into for additional information about collaborations.
+
+## Table of Contents
+
+* [Process](#process)
+* [Git Branches](#git-branches)
+* [Commit Messages](#commit-messages)
+
+## Process
+
+1. Clone this repository
+2. Create a branch for your feature (see [below](#feature-branch-example) for an example)
+3. Add your code
+4. Create a Pull Request
+
+## Git Branches
+
+Below you can find an overview of the branches we are using.
+
+```
+- master (protected)
+- dev (protected)
+- documentation
+- feature
+-- feature/FEATURE-NAME
+--- feature/FEATURE-NAME/FEATURE-PART
```

Comment:
We found that using multiple slashes in a branch name causes issues, should we change these to "feature/featureName-featurePart" or similar?

Reply:
I actually learned that the hard way a little bit earlier so if this is something that's already been discovered by the rest of the team we should change it too.